com.unity.toonshader/Documentation~/sample-instlation.md

```diff
@@ -9,11 +9,6 @@ Unity provides  the **Unity Toon Shader** sample scenes for the three render pip
 1. Set Graphics Pipeline Asset in the [Project Setting window](#https://docs.unity3d.com/2022.2/Documentation/Manual/comp-ManagerGroup.html).  URP samples require `UTS2URPPipelineAsset`. Whereas HDRP samples require  `HDRenderPipelineAsset_UTS`.
 <br/><br/>
 
-| Where you should set `UTS2URPPipelineAsset` or `HDRenderPipelineAsset_UTS` |
```

Comment:
Would it be possible to leave at least this part ?

![image](https://github.com/user-attachments/assets/ef9e3cd6-e4e2-4412-ac08-3f4f73d3fb80)

It may not be so easily noticeable to users.